.circleci/config.yml

```diff
@@ -1,91 +1,43 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
-version: 2
+version: 2.1
+orbs:
+  gem: zfhui/ruby-gem@0.2.1
```

Comment:
May be in the future could we do this like `blinkist/ruby-gem`?

Reply:
Agreed, I think that's Fei's intention.